Create post: Lessons learned from cross-cutting work across BAT (#1574)
DFE-Digital/bat-design-history: app/posts/becoming-a-teacher/2025-04-23-lessons-learned-from-cross-cutting-work-across-BAT.md

## Hypothesis

The hypothesis underpinning this work was that the current process of becoming a trainee teacher is not as efficient as it could be.

This hypothesis is supported by user research insights, which have identified that users, in particular, find the transition from GIT to Find quite jarring.

That insight, in part, informed the decision to redesign the Find home page to align it more closely with GIT and to simplify the process of finding courses.

This work is detailed in this [design history](/find-teacher-training/improving-the-ability-to-browse-courses/).

## What was done

The Find UCD team began by reviewing all evidence, data, insights, and research undertaken by the service teams from 2018 onwards.

They found key insights that supported the hypothesis and, as a result, set about designing a more connected and smoother candidate journey.

The challenge for this work was that the Find UCD team were designing work that impacted all three services: GIT, Find and Apply.

## Challenges

> As the work impacted all three services, all three service teams need to be involved and bought into this work. This required a sign-off from three different Deputy Directors.

Reviewer: ```suggestion
As the work impacted all three services, all three service teams needed to be involved and bought into this work. This required a sign-off from three different Deputy Directors.
```

Whilst this was achieved mid-project, it may have been more desirable that this had been done initially.

## Outputs

> The Find UCD team made several cross-cutting recommendations, which were ultimately parked, as service teams needed to understand how and when to prioritise these work strands into their roadmaps.

Reviewer: What were these cross-cutting recommendations, and can they be documented here for future reference? Is there a separate design history post that captures the information?

Author: These are documented in SharePoint, and I think as they are recommendations that may or may not be taken forward, I am content to leave this out of this DH.

The Find team decided to move forward with candidate accounts on Find, as this was work they could fully own and implement, and would have several clear benefits for users, as well as the Department for Education (DfE) and the initial teacher training (ITT) sector.

## Lessons learned

> After the decision was made to focus on candidate accounts on Find, a [retrospective](https://www.gov.uk/service-manual/agile-delivery/agile-tools-techniques#retrospective-meetings) meeting was held to ensure that all involved could provide feedback on the work, highlight what went well, and flag areas that could be improved for the future.

Reviewer: Are there any lessons learned that the Find team could share in this post or a separate design history?

Author: Yeah, I think I need to make it clear that the recommendations are the key lessons learned. Hopefully have done that.

---
title: Lessons learned from cross-cutting work across the Becoming a teacher service line
description: Lessons learned and recommendations for future cross-cutting work
date: 2025-04-23
tags:
- becoming a teacher
- find
- apply
- Get Into Teaching
related:
  items:
    - text: Get Into Teaching
      href: https://getintoteaching.education.gov.uk/
    - text: Find teacher training courses
      href: https://find-teacher-training-courses.service.gov.uk/

>     - text: Apply for teacher training courses

Reviewer: ```suggestion
    - text: Apply for teacher training
```

      href: https://www.gov.uk/apply-for-teacher-training/
    - text: Improving the ability to browse courses
      href: https://becoming-a-teacher.design-history.education.gov.uk/find-teacher-training/improving-the-ability-to-browse-courses/
    - text: What are self-managing teams
      href: https://www.scrum.org/learning-series/self-managing-teams/what-are-self-managing-teams-
---

> 
> ## Context

Reviewer: ```suggestion
```

> In early 2025, the [Find](https://find-teacher-training-courses.service.gov.uk/) user-centred design (UCD) team were tasked with undertaking a research and design project looking at the candidate journey, from consideration of teaching through to submitting an application for a trainee teacher course.

Reviewer: ```suggestion
In early 2025, the user-centred design (UCD) team at [Find teacher training courses (Find)](https://find-teacher-training-courses.service.gov.uk/) was tasked with undertaking a research and design project examining the candidate journey, from considering teaching through to applying for a trainee teacher course.
```

> This journey, for many candidates, involves moving back and forth from [Get Into Teaching](https://getintoteaching.education.gov.uk/), [Find](https://find-teacher-training-courses.service.gov.uk/) and [Apply](https://www.gov.uk/apply-for-teacher-training)

Reviewer: ```suggestion
This journey, for many candidates, involves moving back and forth from [Get Into Teaching](https://getintoteaching.education.gov.uk/), [Find](https://find-teacher-training-courses.service.gov.uk/) and [Apply for teacher training (Apply)](https://www.gov.uk/apply-for-teacher-training)
```

> ## Hypothesis
> 
> The hypothesis underpinning this work was that the current process of becoming a trainee teacher is not as efficient as it could be.
> 
> This hypothesis is supported by user research insight, which has identified that users, in particular, find the journey of moving from GIT to Find quite jarring.
> 
> That insight, in part, informed the decision to redesign the home page of Find, to align it more with GIT, and to make the process of finding courses less burdensome.
> 
> This work is detailed in this [design history](/find-teacher-training/improving-the-ability-to-browse-courses/).

Reviewer: ```suggestion
## Hypothesis

The hypothesis underpinning this work was that the current process of becoming a trainee teacher is not as efficient as it could be.

This hypothesis is supported by user research insights, which have identified that users, in particular, find the transition from GIT to Find quite jarring.

That insight, in part, informed the decision to redesign the Find home page to align it more closely with GIT and to simplify the process of finding courses.

This work is detailed in this [design history](/find-teacher-training/improving-the-ability-to-browse-courses/).
```

> ## What was done
> 
> The Find UCD team began by reviewing all evidence, data, insights and research that had been undertaken by the service teams, from 2018 onwards.
> 
> They found key insights which supported the hypothesis and as a result set about designing how a more connected and smoother candidate journey may look.
> 
> The challenge for this work, was that the Find UCD team were designing work that impacted all three services: GIT, Find and Apply.

Reviewer: ```suggestion
## What was done

The Find UCD team began by reviewing all evidence, data, insights, and research undertaken by the service teams from 2018 onwards.

They found key insights that supported the hypothesis and, as a result, set about designing a more connected and smoother candidate journey.

The challenge for this work was that the Find UCD team were designing work that impacted all three services: GIT, Find and Apply.
```

> ## Challenges
> 
> As the work impacted all three services, all three service teams need to be sighted and bought-in to this work. This required sign-off from three different Deputy Directors.
> 
> Whilst this was achieved mid-project, it may have been more desirable that this had been done initially.

Reviewer: ```suggestion
## Challenges

As the work impacted all three services, all three service teams need to be involved and bought into this work. This required a sign-off from three different Deputy Directors.

Whilst this was achieved mid-project, it may have been more desirable that this had been done initially.
```

> ## Outputs
> 
> The Find UCD team made several cross-cutting recommendations which were ultimately parked, as service teams needed to understand how and when to prioritise these work strands into their roadmaps.
> 
> The Find team made the decision to move forward with candidate accounts on Find, as this was work that they could fully own and implement, and would have several clear benefits to users, as well as DfE and the initial teacher training (ITT) sector.

Reviewer: ```suggestion
## Outputs

The Find UCD team made several cross-cutting recommendations, which were ultimately parked, as service teams needed to understand how and when to prioritise these work strands into their roadmaps.

The Find team decided to move forward with candidate accounts on Find, as this was work they could fully own and implement, and would have several clear benefits for users, as well as the Department for Education (DfE) and the initial teacher training (ITT) sector.
```

> ## Lessons learned
> 
> After the decision was made to focus on candidate accounts on Find was made, a [retrospective](https://www.gov.uk/service-manual/agile-delivery/agile-tools-techniques#retrospective-meetings) was run to ensure that all involved could feedback on the work, highlight what went well, and flag things that could be improved for the future.

Reviewer: ```suggestion
## Lessons learned

After the decision was made to focus on candidate accounts on Find, a [retrospective](https://www.gov.uk/service-manual/agile-delivery/agile-tools-techniques#retrospective-meetings) meeting was held to ensure that all involved could provide feedback on the work, highlight what went well, and flag areas that could be improved for the future.
```

> ## Recommendations
> 
> We identified several key recommendations which we think are appropriate to be shared across the becoming a teacher service line, as well as within Teacher Services.
> 
> These are:
> 
> - for future cross-service or exploratory work, a consistent scoping document should be used at the outset of the work to capture key goals and outcomes, required resources, involved stakeholders, and guiding principles
> - there should be a clear understanding from the outset of how recommendations will be actioned and prioritised by senior civil servants
> - senior level sign-off and buy-in will be secured as part of the initial scoping
> - teams should be empowered to be [self-managing](https://www.scrum.org/learning-series/self-managing-teams/what-are-self-managing-teams-)
> - design explorations, are by their nature a safe space to experiment, learn, and even fail or pivot
> - there should be a consideration of setting-up an experimental ‘test and learn’ team dedicated to cross-service, with service teams being used on a consultancy basis in the future.

Reviewer: I wonder if the bullets should start with a capital letter, given their current format.

```suggestion
## Recommendations

We identified several key recommendations that are suitable for sharing across the Becoming a Teacher service line, as well as within Teacher Services. These included:

- for future cross-service or exploratory work, a consistent scoping document should be used at the outset of the work to capture key goals and outcomes, required resources, involved stakeholders, and guiding principles
- there should be a clear understanding from the outset of how recommendations will be actioned and prioritised by senior civil servants
- senior-level sign-off and buy-in will be secured as part of the initial scoping
- teams should be empowered to be [self-managing](https://www.scrum.org/learning-series/self-managing-teams/what-are-self-managing-teams-)
- design explorations are, by their nature, a safe space to experiment, learn, and even fail or pivot
- there should be a consideration of setting up an experimental ‘test and learn’ team dedicated to cross-service, with service teams being used on a consultancy basis in the future.
```
